modules/kapacitor-cluster/README.md

```diff
@@ -0,0 +1,127 @@
+https://github.com/gruntwork-io/terraform-aws-influx/tree/master/modules/install-kapacitor) module.
+
+## How do you use this module?
+
+This folder defines a [Terraform module](https://www.terraform.io/docs/modules/usage.html), which you can use in your
+code by adding a `module` configuration and setting its `source` parameter to URL of this folder:
+
+```hcl
+module "kapacitor_cluster" {
+  # TODO: replace <VERSION> with the latest version from the releases page: https://github.com/gruntwork-io/terraform-aws-influx/releases
+  source = "github.com/gruntwork-io/terraform-aws-influx//modules/kapacitor-cluster?ref=<VERSION>"
+
+  # Specify the ID of the Kapacitor AMI. You should build this using the scripts in the install-kapacitor module.
+  ami_id = "ami-abcd1234"
+  
+  # Configure and start Kapacitor during boot. 
+  user_data = <<-EOF
+              #!/bin/bash
+              sudo systemctl start kapacitor
+              EOF
+  
+  # ... See variables.tf for the other parameters you must define for the kapacitor-cluster module
+}
+```
+
+Note the following parameters:
+
+* `source`: Use this parameter to specify the URL of the kapacitor-cluster module. The double slash (`//`) is 
+  intentional and required. Terraform uses it to specify subfolders within a Git repo (see [module 
+  sources](https://www.terraform.io/docs/modules/sources.html)). The `ref` parameter specifies a specific Git tag in 
+  this repo. That way, instead of using the latest version of this module from the `master` branch, which 
+  will change every time you run Terraform, you're using a fixed version of the repo.
+
+* `ami_id`: Use this parameter to specify the ID of an Kapacitor [Amazon Machine Image 
+  (AMI)](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AMIs.html) to deploy on each server in the cluster. You
+  should install Kapacitor the scripts in the 
+  [install-kapacitor](https://github.com/gruntwork-io/terraform-aws-influx/tree/master/modules/install-kapacitor) module.
+  
+* `user_data`: Use this parameter to specify a [User 
+  Data](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/user-data.html#user-data-shell-scripts) script that each
+  server will run during boot. This is where you can use the 
+  [run-kapacitor](https://github.com/gruntwork-io/terraform-aws-influx/tree/master/modules/run-kapacitor) 
+  script to configure and run Kapacitor. 
+
+You can find the other parameters in [variables.tf](variables.tf).
+
+Check out the [examples folder](https://github.com/gruntwork-io/terraform-aws-influx/tree/master/examples) for 
+fully-working sample code.
+
+## Why Kapacitor enterprise?
+
+Unlike the Enterprise edition which distributes the entire Kapacitor deployment across multiple nodes in a cluster,
+the OSS edition is a single binary that can simply be installed on a single instance. This makes a robust module like this one
+quite unnecessary for single instance Kapacitor OSS deployments.
```

Comment:
Should we rephrase this paragraph a bit to say Why Kapitor OSS instead?

or 'When should I not use this module?'

I was a bit confused when I first read it as I believe its **for** the Kapacitor Enterprise module.

Reply:
Noted!